Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -67,11 +72,91 @@ enumerate = false
 ignore_group_members = true
 ```
 
+The SSSD configuration file may be omitted from the CephNFS spec if desired. In this case, Rook will
+not set `/etc/sssd/sssd.conf` in any way. This allows you to manage the `sssd.conf` file yourself
+however you wish. For example, you may build it into your custom Ceph container image, or use the
+[Vault agent injector](https://www.vaultproject.io/docs/platform/k8s/injector) to securely add the
+file via annotations on the CephNFS spec (passed to the NFS server pods).
+
+
+## User authentication
+
+User authentication allows NFS clients and the Rook CephNFS servers to authenticate with each other
+to ensure security.
+
+### Authentication through Kerberos
+
+Kerberos is the authentication mechanism natively supported by NFS-Ganesha. With NFSv4, individual
+users are authenticated and not merely client machines.
+
+#### **Kerberos configuration**
```

Comment:
Since we moved to the new doc rendering, not sure we need the bold on the L4 headers anymore

Reply:
Removed this from all NFS docs.